doc/content/lorawan/classes/_index.md

```diff
@@ -12,13 +12,112 @@ The LoRaWAN specification defines three device types. All LoRaWAN devices must i
+
+*   Environmental monitoring
+*   Animal tracking
+*   Fire detection
+*   Water leakage detection
+*   Earthquake early detection
+*   Location tracking
 
 ## Class B
 
 **Class B** devices extend Class A by adding scheduled receive windows for downlink messages from the server. Using time-synchronized beacons transmitted by the gateway, the devices periodically open receive windows.
 
+Class B end devices have lower latency than the Class A end devices, because they are reachable at preconfigured times and do not need to send an uplink to receive a downlink. The battery life is shorter in Class B than Class A. 
+
+![Class B Receive Windows](../class-B.png)
+
+Some common use cases for Class B end-devices:
+
+*   Smart metering
+*   Temperature reporting
+
 ## Class C
 
 **Class C** devices extend Class A by keeping the receive windows open unless they are transmitting, as shown in the figure below. This allows for low-latency communication but is many times more energy consuming than Class A devices.
 
 ![Class C Receive Windows](../rx-class-c.png)
+
+Class C end devices:
+
+*   Are often mains powered
+*   Have no downlink latency - continuous receive window
+
+The following are some common **Class C** end device **applications**, but there are more:
+
+*   Utility meters
+*   Streetlights
+
+##  Questions
+
+1. Which end device class consumes the lowest power?
+   - <span style="text-decoration:underline;">Class A</span>
+   - Class B
+   - Class C
+   
+   
+2. Which end device class consumes the highest power?
+   - Class A
+   - Class B
+   - <span style="text-decoration:underline;">Class C</span>
+   
+   
+3. Which end device class usually runs on mains power?
+   - Class A
+   - Class B
+   - <span style="text-decoration:underline;">Class C</span>
+   
+   
+4. What does RX1 Delay mean?
+   - <span style="text-decoration:underline;">The delay between the end of the uplink transmission and the start of the RX1 receive window</span>.
+   - The delay between the end of the uplink transmission and the start of the RX2 receive window.
+   - The delay between the end of the RX1 and the end of the RX2 receive windows.
+   - The delay between the start of the RX1 and the start of the RX2 receive windows.
+   
+   
+5. Which device class has the lowest downlink latency?
+   - Class A
+   - Class B
+   - <span style="text-decoration:underline;">Class C</span>
+   
+   
+6. Which device class has the highest downlink latency?
+   - <span style="text-decoration:underline;">Class A</span>
+   - Class B
+   - Class C
+   
+   
+7. Which device class is synchronized to the network using periodic beacons?
+   - Class A
+   - <span style="text-decoration:underline;">Class B</span>
+   - Class C
+   
+   
+8. Which device class only can receive a downlink message after sending an uplink message?
+   - <span style="text-decoration:underline;">Class A</span>
+   - Class B
+   - Class C
+   
+   
+9. The recommended value for RX1 Delay for all regions is
```

Comment:
We should probably provide the info that The Things Stack default is now 5 seconds, not 1